jest.config.json

```diff
@@ -2,12 +2,20 @@
   "clearMocks": true,
   "moduleDirectories": ["node_modules", "src"],
   "moduleFileExtensions": ["js", "jsx"],
+  "moduleNameMapper": {
+    "^.+\\.(gif|jpg|jpeg|png)$": "<rootDir>/mocks/fileMock.js",
+    "^.+\\.css$": "<rootDir>/mocks/fileMock.js"
+  },
   "rootDir": "src",
+  "setupFilesAfterEnv": [
+    "@testing-library/jest-dom"
+  ],
   "testEnvironment": "jsdom",
   "testRegex": "((\\.|/*.)(spec))\\.js?$",
   "transform": {
     "^.+\\.(js|jsx)$": ["babel-jest", {"rootMode": "upward"}],
-    "^.+\\.html$": "<rootDir>/spec/html-loader.cjs"
+    "^.+\\.html$": "jest-html-loader",
+    "^.+\\.(ts|tsx)$": "ts-jest"
```

Comment:
This was added to try and fix the issue with the OpenLayers dependency not working, but didn't solve it.  However, it's probably still worth leaving here as it will help with other typescript imports if needed.